src/components/HURUmap/ChartFilter/index.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import ChartFilter from "@/pesayetu/components/HURUmap/ChartFilter";
+
+export default {
+  title: "Components/HURUmap/ChartFilter",
+  argTypes: {},
+};
+
+const Template = (args) => <ChartFilter {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  selected: "All Values",
```

Comment:
By setting this, I can't select any other value

Reply:
Fixed